metrictank.go

```diff
@@ -384,6 +385,8 @@ func main() {
 	/***********************************
 		Initialize our MetricIdx
 	***********************************/
+	cache := cache.NewCCache()
+	metrics = mdata.NewAggMetrics(store, chunkSpan, numChunks, chunkMaxStale, metricMaxStale, ttl, gcInterval, finalSettings)
```

Comment:
this doesn't look right. why is the diff adding a NewAggMetrics() line but not removing one?